src/main/java/com/google/sps/Business.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps;
+
+import java.net.URL;    
+import java.util.Comparator;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+* Holds all relevant data to a Business on the site.
+*/
+public final class Business {
+
+  private String name;
+  private List<String> categories = new ArrayList<String>();
+  private float aggregatedRating;
+  private float addressLng;
+  private float addressLat;
+  private String address;
+  private String logoBlobstoreUrl;
+  private List<String> photoBlobstoreUrlList = new ArrayList<String>();
+  private String description;
+  private String menuUrl;
+  private String orderInformation;
+  private String contactUrl;
+  private String websiteUrl;
+  private List<String> reviewDatastoreKeyList = new ArrayList<String>();
```

Comment:
Just curious, how do you determine the order being added? I think its better if we group it tgt by data structure to be neater?
Also, you dont have to initialize the lists as you are assigning it directly from the constructor. `private List<String> categories;` will work fine

Reply:
I ordered it in importance as well as structure on the product page (so all the contact, business, review information is further down, and address info is grouped together) I do think by data structure is neater, but I don't see major differences either way. thoughts?